data/apdb-ap-pipe-schema-extra.yaml

```diff
@@ -612,10 +612,6 @@ columns:
   type: CHAR
   nullable: false
   description: Name of stored filter.
-- name: filterId
-  type: INT
-  nullable: false
-  description: id number of stored filter.
```

Comment:
Even if it wasn't being used, I think any kind of schema change in the APDB could break things, so is @morriscb ok with this?

Reply:
On slack, @morriscb said "The later isn’t used anymore I don’t think so that’s fine": https://lsstc.slack.com/archives/C2B6X08LS/p1608158086072200